portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssertBlogs.testcase

```diff
@@ -36,6 +36,54 @@ definition {
 		ACUtils.tearDownAC();
 	}
 
+	@description = "View all blogs in the property in assets"
+	@priority = "5"
+	test ViewAllBlogsShownInAssetList {
+		ACUtils.createBlogsAndAddToPage(entryTitle = "Test 2");
+
+		ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		for (var n : list "1,2") {
+			Navigator.gotoSitePage(
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			BlogsNavigator.gotoPG(entryTitle = "Blogs AC Title");
+
+			Navigator.gotoSitePage(
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			BlogsNavigator.gotoPG(entryTitle = "Test 2");
+		}
+
+		Pause(locator1 = "10000");
```

Comment:
Hi @joshitagaki , did you check if the test works without this `Pause(locator1 = "10000");`? I don't know if we need it here since we are only viewing the blog and not adding an entry/submitting a form...